pykern/pkcli/ci.py

```diff
@@ -73,6 +64,48 @@ def run():
     from pykern.pkcli import fmt, test
     from pykern import pkio
 
+    check_eof_newline()
     check_prints()
     fmt.diff(pkio.py_path())
     test.default_command()
+
+
+def _check_files(case, check_file):
+    def _error(m):
+        pkcli.command_error("{}: {}", case, m)
+
+    d = PKDict(
+        check_eof_newline=PKDict(
+            exclude_files=re.compile(
+                r"/static/js/ext/|node_modules/|^run/|tests/|^venv/"
```

Comment:
A little documentation here might help. It's ok to mention that static/js/ext is where Sirepo stores 3rd party library files. I think we may want to ignore "^tests/" not just "tests/". Scientists use the word "tests" to mean conditions to test. Narrowing here would be the comparison is useful. Similarly, `^node_modules/` is probably a good idea.

Reply:
We need to include react/node_modules so I think it makes sense to keep `node_modules/` as it is.
a9e80a0beda32d4163def682c4cffe18637e6076